terraform/modules/hub/files/cloud-init/prometheus.sh

```diff
@@ -68,59 +68,14 @@ run-until-success apt-get install --yes docker.io
 cat <<EOF > /etc/systemd/system/docker.service.d/override.conf
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd --log-driver journald --dns 10.0.0.2
+ExecStart=/usr/bin/dockerd --log-driver awslogs --log-opt awslogs-region=eu-west2 --log-opt awslogs-group ${cloudwatch_log_group} --dns 10.0.0.2
 EOF
 
 # Reload systemctl daemon to pick up new override files
 systemctl stop docker
 systemctl daemon-reload
 systemctl enable --now docker
 
-# Journalbeat for log shipping
-echo 'Installing and configuring journalbeat'
-(
-elastic_beats="artifacts.elastic.co/downloads/beats"
-mkdir -p /tmp/journalbeat
-cd /tmp/journalbeat
-
-cat <<EOF > journalbeat-oss-6.8.3-amd64.deb.sha512
-685e571638a3422e8b1c6f6aa7c15db8bf8fa9b91ecfedb4ce7c26dedc418e90b558a37711af2a547cb5025de17361d2fed1042be2d0871d22ec78037f7225a6  journalbeat-oss-6.8.3-amd64.deb
-EOF
-
-curl --silent --fail \
-     -L -O \
-     "https://$elastic_beats/journalbeat/journalbeat-oss-6.8.3-amd64.deb"
-
-sha512sum -c journalbeat-oss-6.8.3-amd64.deb.sha512
-dpkg -i journalbeat-oss-6.8.3-amd64.deb
-)
-
-cat <<EOF > /etc/journalbeat/journalbeat.yml
-http.enabled: true
-
-journalbeat.inputs:
-- paths: []
-  seek: cursor
-
-logging.level: warning
-logging.to_files: false
-logging.to_syslog: true
-logging.json: true
-
-processors:
-- add_cloud_metadata: ~
-
-output.elasticsearch:
-  hosts: ["https://${logit_elasticsearch_url}:443"]
-  headers:
-    Apikey: ${logit_api_key}
-EOF
-# It seems that journalbeat is very unhappy if docker isn't running
-# when it starts, so let's update the systemd unit to reflect that
-sed -i -e 's/Wants=.*/& docker.service/' /lib/systemd/system/journalbeat.service
-sed -i -e 's/After=.*/& docker.service/' /lib/systemd/system/journalbeat.service
-systemctl enable --now journalbeat
-
```

Comment:
now that we've got rid of journalbeat, we could run prometheus on `m6g` instances (slight troll, but I think the issues I found in https://github.com/alphagov/prometheus-aws-configuration-beta/pull/400 are not a concern in this PR, except the only-two-AZs one)